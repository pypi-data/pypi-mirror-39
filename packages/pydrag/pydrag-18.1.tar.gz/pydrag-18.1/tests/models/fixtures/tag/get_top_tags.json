{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.1"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&method=tag.getTopTags&num_res=10&offset=20"
            },
            "response": {
                "body": {
                    "string": "{\"toptags\":{\"@attr\":{\"offset\":20,\"num_res\":10,\"total\":2693772},\"tag\":[{\"name\":\"black metal\",\"count\":808560,\"reach\":60919},{\"name\":\"dance\",\"count\":789981,\"reach\":131041},{\"name\":\"80s\",\"count\":768919,\"reach\":99629},{\"name\":\"Progressive rock\",\"count\":705995,\"reach\":94830},{\"name\":\"death metal\",\"count\":696989,\"reach\":70446},{\"name\":\"hardcore\",\"count\":679448,\"reach\":96344},{\"name\":\"british\",\"count\":679004,\"reach\":92377},{\"name\":\"heavy metal\",\"count\":676538,\"reach\":89132},{\"name\":\"soul\",\"count\":659170,\"reach\":99192},{\"name\":\"chillout\",\"count\":642670,\"reach\":102845}]}}"
                },
                "headers": {
                    "Access-Control-Allow-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS, PUT"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "568"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sun, 11 Nov 2018 15:56:16 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ],
                    "TSVC-Time-Spent": [
                        "0.000"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
