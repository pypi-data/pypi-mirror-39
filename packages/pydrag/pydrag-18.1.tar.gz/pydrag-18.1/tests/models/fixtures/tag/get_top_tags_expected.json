{
    "data": [
        {
            "count": 808560,
            "name": "black metal",
            "reach": 60919
        },
        {
            "count": 789981,
            "name": "dance",
            "reach": 131041
        },
        {
            "count": 768919,
            "name": "80s",
            "reach": 99629
        },
        {
            "count": 705995,
            "name": "Progressive rock",
            "reach": 94830
        },
        {
            "count": 696989,
            "name": "death metal",
            "reach": 70446
        },
        {
            "count": 679448,
            "name": "hardcore",
            "reach": 96344
        },
        {
            "count": 679004,
            "name": "british",
            "reach": 92377
        },
        {
            "count": 676538,
            "name": "heavy metal",
            "reach": 89132
        },
        {
            "count": 659170,
            "name": "soul",
            "reach": 99192
        },
        {
            "count": 642670,
            "name": "chillout",
            "reach": 102845
        }
    ],
    "limit": 10,
    "page": 2,
    "total": 2693772
}
